src/Simplex/Chat/Controller.hs

```diff
@@ -438,10 +445,10 @@ data ChatCommand
 data ChatResponse
   = CRActiveUser {user :: User}
   | CRUsersList {users :: [UserInfo]}
-  | CRChatStarted
-  | CRChatRunning
-  | CRChatStopped
-  | CRChatSuspended
+  | CRChatStarted {_nullary :: Maybe Int}
+  | CRChatRunning {_nullary :: Maybe Int}
+  | CRChatStopped {_nullary :: Maybe Int}
+  | CRChatSuspended {_nullary :: Maybe Int}
```

Comment:
`Maybe Void`?

Reply:
It wouldn't. Void makes `Just` unrepresentable, making Nothing the only option (to be eliminated later).